src/Symfony/Component/Form/Tests/Extension/Validator/DataTransformer/ViewObjectTransformer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Symfony\Component\Form\Tests\Extension\Validator\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+* Transforms Object to array for view
+*
+* @author Charles J. C. Elling <charles@denumeris.com>
+*/
+class ViewObjectTransformer implements DataTransformerInterface {
```

Comment:
Because  we added a view transformer that transforms a instance of _class_name_ to a _a(n) array_, to pass the validation "data_class" needs to be _null_.